azure-pipelines-full.yml

```diff
@@ -30,6 +36,16 @@ jobs:
   pool:
     vmImage: 'macOS-10.14'
   steps:
+  - bash: sudo $AGENT_HOMEDIRECTORY/scripts/select-xamarin-sdk.sh $(MONO_VERSION)
```

Comment:
Is it possible to extract those scripts in an other file somehow to avoid repeating them in both yml files?

Reply:
Yes we can have a 'default-pipeline' where this is set. Saw this in some other examples.

Reply:
Thought with the default-pipeline it can work but because I wanted to set the Versions in the template it won't. I'll try it with a shell-script - should work.